src/lookup.py

```diff
@@ -120,30 +126,40 @@ def lookup(self):
 
 
 class StageLookup(Lookup):
-    def __init__(self, version, subVer, parent):
-        super().__init__("Stages", version, subVer)
+    def __init__(self, version, parent):
+        self.xml = None
+        super().__init__("Stages", version)
         self.parent = parent
 
+    def count(self):
+        if self.xml:
+            return len(self.xml)
+        else:
+            return 0
+
     def loadXML(self, root, resourcePath="", modName="Basement Renovator"):
         stageList = root.findall("stage")
         if not stageList:
             return
 
+        initialLoad = False
+        if self.xml is None:
+            self.xml = root
+            initialLoad = True
+
         def mapStage(stage):
             name = stage.get("Name")
             if name is None:
                 print("Tried to load stage, but had missing name!", str(stage.attrib))
                 return None
 
-            print("Loading stage:", str(stage.attrib))
```

Comment:
I removed this + the room type log later because they created a lot of unhelpful clutter in the log. Wanted to switch them out in favor of more error detection printing. The new "Loaded N Stages / Entities / RoomTypes" log acts as a pseudo-replacement for this, with regards to indicating that they've loaded successfully and how many.